frontends/web/package.json

```diff
@@ -19,11 +19,11 @@
   },
   "dependencies": {
     "@zxing/library": "0.18.3",
-    "i18next": "20.6.0",
+    "i18next": "21.6.3",
```

Comment:
There are a [few breaking changes](https://github.com/i18next/i18next/blob/master/CHANGELOG.md#2100), I believe one of them affects us:

- afaik we only use it here https://github.com/digitalbitbox/bitbox-wallet-app/blob/243f3e0d47e6906c66c5aed8f7cae6616d93837a/frontends/web/src/locales/en/app.json#L956

> introduce new v4 json format (intl plural forms with ordinal support), simplifyPluralSuffix not used anymore

new with v4
- https://www.i18next.com/translation-function/plurals

Reply:
I prepared a PR to rename the keys but it is not pushed to locize yet:

https://github.com/digitalbitbox/bitbox-wallet-app/pull/1568

Reply:
that's an annoying breaking change that was totally unnecessary. could have kept this suffix for backwards compatibility :(